### ISSUE TITLE
I also received this on GKE with all of my containers failing in the Dockerfile:

### ISSUE THREAD

**Issue by CryptoGodfatherVA** [armosec/kubescape#20]
I also received this on GKE with all of my containers failing in the Dockerfile:
`RUN chown -R node: /tmp /app
USER node
RUN mkdir /home/node/.config/
EXPOSE 3000`

and in my deployment manifest 

```---
apiVersion: v1
kind: Service
metadata:
  annotations:
  name: service-name
spec:
  ports:
  - port: 3000
    protocol: TCP
    targetPort: 3000
  selector:
    app: service-name
  sessionAffinity: None
  type: ClusterIP

---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: service-name
  labels:
    app: service-name
spec:
  progressDeadlineSeconds: 600
  replicas: {{ .Values.api_service.replicas }}
  revisionHistoryLimit: 10
  selector:
    matchLabels:
      app: service-name
  strategy:
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
    type: RollingUpdate
  template:
    metadata:
      annotations:
        rollme: {{ randAlphaNum 5 | quote }}
      labels:
        app: service-name
    spec:
      topologySpreadConstraints:
      - maxSkew: 1
        topologyKey: node
        whenUnsatisfiable: ScheduleAnyway
        labelSelector:
          matchLabels:
            app: service-name
      containers:
      - env:
        - name: ENVIRONMENT
          value: prod
        image: {{IMGAGE_REPOSITORY}}:{{ .Values.api_service.docker_tag }}
        imagePullPolicy: Always
        resources:
          requests:
            cpu: 1
        livenessProbe:
          failureThreshold: 3
          httpGet:
            path: /health-check
            port: 3000
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 30
          successThreshold: 1
          timeoutSeconds: 1
        name: service-name
        ports:
        - containerPort: 3000
          protocol: TCP
        readinessProbe:
          failureThreshold: 3
          httpGet:
            path: /health-check
            port: 3000
            scheme: HTTP
          initialDelaySeconds: 5
          periodSeconds: 10
          successThreshold: 1
          timeoutSeconds: 5
        terminationMessagePath: /dev/termination-log
        terminationMessagePolicy: File
        volumeMounts:
        - name: gcp-credentials-volume
          mountPath: /secrets/gcp
          readOnly: true
      dnsPolicy: ClusterFirst
      restartPolicy: Always
      schedulerName: default-scheduler
      terminationGracePeriodSeconds: 30
```

_Originally posted by @bluehawk27 in https://github.com/armosec/kubescape/issues/19#issuecomment-904073840_

**Comment by bluehawk27** [comment]
Deployment manifest and docker file running as NODE user is failing the non-root check

**Comment by dwertent** [comment]
@bluehawk27 @CryptoGodfatherVA Thank you for pointing out this issue. We will update once we deliver the fix.

**Comment by dwertent** [comment]
Closed since it's a duplicate of #19.

We will work on improving this test.
